diff --git a/plugins/bundler/bundler.plugin.zsh b/plugins/bundler/bundler.plugin.zsh
index 382a1a4..e8828ac 100644
--- a/plugins/bundler/bundler.plugin.zsh
+++ b/plugins/bundler/bundler.plugin.zsh
@@ -91,8 +91,10 @@ _binstubbed() {
 _run-with-bundler() {
   if _bundler-installed && _within-bundled-project; then
     if _binstubbed $1; then
+      print 'command executed: ./bin/'$@
       ./bin/$@
     else
+      print 'command executed: bundle exec' $@
       bundle exec $@
     fi
   else
diff --git a/plugins/rails/rails.plugin.zsh b/plugins/rails/rails.plugin.zsh
index 8af1d63..628b492 100644
--- a/plugins/rails/rails.plugin.zsh
+++ b/plugins/rails/rails.plugin.zsh
@@ -1,5 +1,6 @@
 function _rails_command () {
   if [ -e "bin/rails" ]; then
+    print 'command executed: bin/rails '$@
     bin/rails $@
   elif [ -e "script/rails" ]; then
     ruby script/rails $@
@@ -12,6 +13,7 @@ function _rails_command () {
 
 function _rake_command () {
   if [ -e "bin/rake" ]; then
+    print 'command executed: bin/rake '$@
     bin/rake $@
   else
     command rake $@
